content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -25,15 +56,20 @@ Search for QuestionnaireResponses that meet supplied query parameters:
 
     GET /QuestionnaireResponse?:parameters
 
+_Implementation Notes_
+
+* QuestionnaireResponses are sometimes system generated without any answers. These may be updated with the correct responses at any time.
+
 ### Authorization Types
 
 <%= authorization_types(provider: true, patient: true, system: true) %>
 
 ### Parameters
 
- Name      | Required? | Type          | Description
------------|-----------|---------------|--------------------------------------------------------
- `_id`     | Yes       | [`token`]     | The logical resource id associated with the resource.
+ Name      | Required?         | Type          | Description
+-----------|------------------ |---------------|--------------------------------------------------------------------------------------------
+ `_id`     | This or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient` | This or `_id`     | [`reference`] | The subject (Patient) that the questionnaire response is about. Example: `patient=12345`
```

Comment:
is reference supposed to link to something?

Reply:
Yes indeed, fixed: 77a659c31a4f0f1174209ad3ead3c07076ea9d13